css/testimonials.css

```diff
@@ -0,0 +1,82 @@
+#client {
+    display: inline-block;
+    width: 100%;
+    height: 550px;
+    text-align: center;
+    background-image: linear-gradient(rgba(0, 0, 0, 0.7), rgba(0, 0, 0, 0.7)),url(../img/client.jpg);
+    background-attachment: fixed;
+    background-repeat: no-repeat;
+    background-position: center;
+    position: relative;
+}
+    .testimonial {
+        display: inline-block;
+        position: absolute;
+        width: 50%;
+        margin: 0;
+        top: 50%;
+        left: 50%;
+        transform: translate(-50%, -50%);
+        text-align: center;
+        background-color: #00000065;
+    }
+        .testimonial h6 {
+            display: inline-block;
+            text-transform: uppercase;
+            color: var(--purple-color);
+            font-size: 14px;
+            font-family: 'Montserrat', sans-serif;
+            font-weight: 400;
+            width: 100%;
+            margin-top: 35px;
+        }
+        .stars {
+            display: inline-block;
+            width: 100%;
+        }
+            .testimonial .fa-star {
+                display: inline-block;
+                font-size: 15px;
+                color: #FFD600;
+                margin: 25px 0px;
+            }
+        .testimonial p {
+            display: inline-block;
+            font-size: 16px;
+            color: #cccccc;
+            width: 80%;
+            line-height: 32px;
+        }
+        .testimonial h5 {
+            color: var(--greyHero-color);
+            text-transform: uppercase;
+            font-family: 'Montserrat', sans-serif;
+            font-weight: 400;
+            font-size: 15px;
+            margin-top: 30px;
+            margin-bottom: 10px;
+        }
+        .testimonial h4 {
+            color: #cccccc;
+            text-transform: uppercase;
+            font-family: 'Montserrat', sans-serif;
+            font-weight: 400;
+            font-size: 16px;
+            margin-bottom: 35px;
+        }
+    .dots {
+        display: inline-block;
+        width: 100%;
+        text-align: center;
+        position: absolute;
+        bottom: 70px;
+        right: 0;
+    }
+        .dot {
+            display: inline-block;
+            width: 15px;
+            height: 4px;
+            margin: 5px;
+            background-color: #D6D6D6;
+            border-radius: 30px;
+        }    
```

Comment:
"No newline at end of file".
What's the significance of the message and what's it trying to tell us?

Reply:
[https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file](url)

pasirodo reikia palikt naują eilutę kodo gale, priežastys nuorodoje.